src/Core/Subscriptions.Redis/RedisSubscriptionServiceCollectionExtensions.cs

```diff
@@ -9,17 +9,17 @@ public static class RedisSubscriptionServiceCollectionExtensions
     {
         public static IServiceCollection AddRedisSubscriptionProvider(
```

Comment:
We should also preserve the old overloads... the old overloads could use the new ones underneath.

Reply:
done